slice-core-api/src/main/java/com/cognifide/slice/api/injector/InjectorConfig.java

```diff
@@ -89,8 +95,8 @@ public String getBundleNameFilter() {
 		return bundleFilter;
 	}
 
-	public String getApplicationPath() {
-		return applicationPath;
+	public List<String> getApplicationPaths() {
```

Comment:
The reason for the code not being compatible is here. We'd need to keep the String getApplicationPath method. However, you can safely add a new method List getApplicationPaths() and use it in InjectorHierarchy. The getApplicationPath should then be made deprecated and the package-info should be bumped to 4.3.0. Does it make sense?